eqcorrscan/tests/tutorials_test.py

```diff
@@ -17,12 +18,19 @@
 from eqcorrscan.core.match_filter import read_detections
 
 
+slow = pytest.mark.skipif(
```

Comment:
This is so that we don't always have to run the tutorial tests - they rely on long data downloads, which appveyor is not so great at handling.